src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -86,12 +86,10 @@ async function loadDrupal(buildOptions) {
 
     console.timeEnd(drupalTimer);
 
-    if (buildOptions.buildtype === ENVIRONMENTS.LOCALHOST) {
-      const serialized = Buffer.from(JSON.stringify(drupalPages, null, 2));
-      fs.ensureDirSync(buildOptions.cacheDirectory);
-      fs.emptyDirSync(path.join(buildOptions.cacheDirectory, 'drupalFiles'));
-      fs.writeFileSync(drupalCache, serialized);
-    }
+    const serialized = Buffer.from(JSON.stringify(drupalPages, null, 2));
+    fs.ensureDirSync(buildOptions.cacheDirectory);
+    fs.emptyDirSync(path.join(buildOptions.cacheDirectory, 'drupalFiles'));
+    fs.writeFileSync(drupalCache, serialized);
```

Comment:
Since it's become somewhat common to build continuously with a certain build-type, it makes sense that we begin caching the Drupal for all environment instead of only localhost.

Reply:
I think we'd want to make sure that Jenkins builds aren't using the cache. There's some workspace state that's cached between builds. I think it's just node_modules, but it could be more than that.

Reply:
Oh good call. I think we can just add the `--pull-drupal` flag into the Jenkinsfile to make sure we pull latest every time?

Reply:
That, or a `--drupal-cache` flag or something.